python-regression/tests/features/steps/api_test_steps.py

```diff
@@ -228,3 +233,60 @@ def check_node_sync(step, node, milestone):
         "Latest Solid Milestone {} on {} is not the expected {}".format(latestSolidMilestone, node, milestone)
 
 
+@step(r'A transaction is issued on "([^"]+)" with:')
+def issue_a_transaction(step, node):
+    """
+    Issues a single transaction
+
+    :param node: The identifier for the node (ie nodeA)
+    """
+    world.config['nodeId'] = node
+    api = api_utils.prepare_api_call(node)
+
+    transaction = transactions.evaluate_and_send(api, "", step.hashes)
+    transaction_hash = Transaction.from_tryte_string(transaction.get('trytes')[0]).hash
+    world.responses['transaction_hash'] = transaction_hash
+
+
+@step(r'"(\d+)" milestones are issued on "([^"]+)"')
+def issue_multiple_milestones(step, num_milestones, node):
+    """
+    Issues num_milestones milestones
+
+    :param num_milestones: The number of milestones to issue
+    :param node: The identifier for the node (ie nodeA)
+    """
+    address = static_vals.TEST_BLOWBALL_COO
+    api = api_utils.prepare_api_call(node)
+    seed = transactions.check_for_seed(step.hashes)
+
+    #Latest milestone is 8412
+    for iteration in range(int(num_milestones)):
+        api = api_utils.prepare_api_call(node, seed=seed)
+        logger.info('Issuing milestone {}'.format(iteration + 1))
+        milestones.issue_milestone(address, api, iteration + 8413)
+        logger.info("sleeping 3 sec")
+        sleep(int(3))
+
+    logger.info("milestones generated")
```

Comment:
This isn't very reusable as written, I would suggest taking a look at my spent addresses test and adding the step used in that PR for issuing the next milestones. That way it isn't reliant on the predefined milestone index and can be used in multiple tests. 
https://github.com/iotaledger/iri/pull/1489/files#diff-633b5f8f41bcc2acc29f24c7c4c4e18fR174

Reply:
Should this PR wait for PR #1489 to be merged so that it uses your `@step(r'the next (\d+) milestones are issued')`step?